cmd/quill/cli/cli.go

```diff
@@ -76,5 +76,7 @@ func New(version clio.Version) *cobra.Command {
 	root.AddCommand(extract)
 	root.AddCommand(p12)
 
+	root.Example = app.SummarizeConfig(root)
```

Comment:
may want to add a comment for future devs about why this is here instead of the in the root constructor (to accumulate configurations across all commands while wiring)